app/src/main/java/com/amazon/connect/chat/androidchatexample/views/ChatComponents.kt

```diff
@@ -45,7 +46,7 @@ fun ChatMessageView(
                     if (transcriptItem.attachmentId != null) {
                         AttachmentMessageView(transcriptItem, viewModel, recentOutgoingMessageID, onPreviewAttachment)
                     } else {
-                        SenderChatBubble(transcriptItem, recentOutgoingMessageID)
+                        SenderChatBubble(transcriptItem, recentOutgoingMessageID, viewModel)
```

Comment:
Do we need to pass in the entire `viewModel`?  Could we just pass the retry function?

Reply:
You're right we don't need to pass the entire viewModel. Will update